che-editors.yaml

```diff
@@ -344,9 +344,15 @@ editors:
       - id: init-container-command
         apply:
           component: che-code-injector
+      - id: init-che-code-command
+        exec:
+          component: che-code-runtime-description
+          commandLine: 'nohup /checode/entrypoint-volume.sh > /checode/entrypoint-logs.txt 2>&1 &'
     events:
       preStart:
         - init-container-command
+      postStart:
+        - init-che-code-command
```

Comment:
The main idea is - initialization of che-code on the `post-start` event vs execution entrypoint as a command to avoid overriding the UDI entrypoint.

I've tested creation of a workspace for the VS Code editor **without** Dashboard usage and I can confirm that the UDI entrypoint is not overridden anymore if I use these changes for the description of the editor.

At the same time there is still a problem when I create a workspace from the dashboard. So - I guess:
- logic on the DWO side works correctly
- we still need some changes on the dashboard side

I guess resolving [this issue](https://github.com/eclipse/che/issues/21738) should fix the problem on the dashboard side.
So, let's keep the PR still as a draft.

Reply:
I tested it using [this sample](https://github.com/RomanNikitenko/devfile-stacks-nodejs-react/tree/post-start), it contains [the same changes](https://github.com/RomanNikitenko/devfile-stacks-nodejs-react/blob/1a5d1ab1eb84e988727cd8df4d10afc10db080a4/.che/che-editor.yaml#L21-L29) for the editor description.

It works well after merging [the corresponding changes](https://github.com/eclipse-che/che-dashboard/pull/716) on the dashboard side.